solutions/mysite/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> Mysite</title>
+       <link rel="stylesheet" href = "style.css">
+    </head>
+    <body>
+    <img src = "https://upload.wikimedia.org/wikipedia/commons/5/5b/Ultraviolet_image_of_the_Cygnus_Loop_Nebula_crop.jpg"></img>
```

Comment:
Also, remind me to talk about hot-linking (look it up if you're curious) next class. There is a better way to link images in your web pages.